frontend/src/metabase-lib/metadata.ts

```diff
@@ -202,3 +202,7 @@ export function fromLegacyColumn(
 ): ColumnMetadata {
   return ML.legacy_column__GT_metadata(query, stageIndex, columnOrField);
 }
+
+export function queryDisplayInfo(query: Query): QueryDisplayInfo {
+  return displayInfo(query, -1, query);
```

Comment:
We should not have 2 functions that do the same thing.
I'd remove the `QueryDisplayInfo`-related overload from `displayInfo`, use `ML.display_info` here directly, and move the comment about the first two arguments here.